### DISCUSSION TITLE
android 14 and xournalpp mobile

### DISCUSSION THREAD

**Discussion by pearcedotsgatgmaildotcom** [xournalpp/xournalpp#5453] [Q&A]
Hi
The apk on google play does not install for android 14 .. could someone talk me through compiling or make an apk which is suitable please? 
regards
Stephen'

**Comment by rolandlo** [comment]
Please raise an issue at the Xournalpp mobile issue tracker https://gitlab.com/TheOneWithTheBraid/xournalpp_mobile/-/issues (but notice that currently nobody maintains Xournalpp mobile).
